search/lib/cmr.js

```diff
@@ -42,6 +42,19 @@ async function cmrSearch (path, params) {
   return axios.get(url, { params, headers: DEFAULT_HEADERS });
 }
 
+/**
+ * Query a CMR Search endpoint with optional parameters using POST.
+ * @param {string} path CMR path to append to search URL (e.g., granules.json, collections.json)
+ * @param {object} params Set of CMR parameters
+ */
+async function cmrSearchPost (path, params) {
```

Comment:
If this is working for all CMR queries then we can replace all `cmrSearch` calls with `cmrSearchPost` (or better, just update `cmrSearch` to use POST)

Reply:
The all the parameters passed to cmrSearch needs to be changed to UrlSearchParams.

Reply:
Ok, then we should leave this as is for now.
Allowing for multiple datetime and intersects parameters is an interesting use case however, I will bring this back to STAC and file an issue for it. May be worth implementing.